setup.py

```diff
@@ -50,6 +50,25 @@ def installPackage(package: str):
 EIP6110 = 'eip6110'
 WHISK = 'whisk'
 
+PREVIOUS_FORK_OF = {
+    PHASE0: None,
+    ALTAIR: PHASE0,
+    BELLATRIX: ALTAIR,
+    CAPELLA: BELLATRIX,
+    DENEB: CAPELLA,
+    EIP6110: DENEB,
+    WHISK: CAPELLA,
+}
+
+ALL_FORKS = list(PREVIOUS_FORK_OF.keys())
+
+IGNORE_SPEC_FILES = [
+    "specs/phase0/deposit-contract.md"
+]
+
+EXTRA_SPEC_FILES = {
+    BELLATRIX: "sync/optimistic.md"
```

Comment:
Used a string since the file accumulator below is not a List. If there are multiple just do a multiline like in 

https://github.com/ethereum/consensus-specs/blob/efd2429b3f14f7bccaad342a75fdf843b21c0951/setup.py#L1072-L1078

Reply:
I was thinking about changing how we append `EXTRA_SPEC_FILES[fork]` together. But it's fine now; we can figure it out when we have multiple extra spec files of one fork.